pkg/scheduler/framework/plugins/volumezone/volume_zone.go

```diff
@@ -116,16 +118,28 @@ func (pl *VolumeZone) PreFilter(ctx context.Context, cs *framework.CycleState, p
 	return nil, nil
 }
 
-func (pl *VolumeZone) getPVbyPod(ctx context.Context, pod *v1.Pod) ([]pvTopology, *framework.Status) {
-	logger := klog.FromContext(ctx)
+// isWaitForFirstConsumer confirms whether storageClass's volumeBindingMode is VolumeBindingWaitForFirstConsumer or not.
+func (pl *VolumeZone) isWaitForFirstConsumer(scName string) (bool, *framework.Status) {
```

Comment:
I guess the change around isWaitForFirstConsumer and getPVbyPod isn't required in this QHint.

Reply:
I think getPVbyPod's change is fine because getPVbyPod and IsPVCRequestedFromPod has duplicated logic.